### ISSUE TITLE
frida-gadget import

### ISSUE THREAD

**Issue by indexofrefraction** [frida/frida#2602]
hi,

is it possible to import with frida-gadget ?
and if so, how?

```
import('./module.js').then(module => {
	// ... do something
}).catch( error => {
	console.log(error); // ReferenceError: could not load module '/module.js'
});
```

note: this is not a typo : './module.js' -->  '/module.js' ?!
the dot gets swallowed and it is impossible to specify a file in the same path

**Comment by indexofrefraction** [comment]
Any answer here please? Is Frida able to import code pieces or do you have to have all inside one file ?

**Comment by indexofrefraction** [comment]
build and **import a toolset** .... dreaming  (^_ -)

**Comment by j4k0xb** [comment]
no. use https://github.com/frida/frida-compile or bundle it yourself with esbuild/swc
